### PR TITLE
Update babel and webpack to fix the build

### PR DESCRIPTION
**Pull request by mcastany**
### Description

* Updated dependencies to start fix the build that was flaky
* Require changes to match the webpack changes

No changes to functionality are expected

### CODE REVIEW

File: build/webpack/config.prod.js [hunk 1]
```diff
@@ -12,43 +12,44 @@ config.profile = false;
 const version = process.env.EXTENSION_VERSION || project.version;
 
 // Build output, which includes the hash.
```
Comment (6footGeek) [inline build/webpack/config.prod.js:14]: nit: hash no longer needed - update comment

File: package-lock.json [hunk 2]
```diff
@@ -1516,1568 +1516,3910 @@
...
-      "resolved": "https://registry.npmjs.org/@types/babel-types/-/babel-types-7.0.4.tgz",
-      "integrity": "sha512-WiZhq3SVJHFRgRYLXvpf65XnV6ipVHhnNaNvE8yCimejrGglkg38kEj0JcizqwSHxmPSjcTlig/6JouxLGEhGw=="
-    },
-    "@types/babylon": {
-      "version": "6.16.3",
-      "resolved": "https://registry.npmjs.org/@types/babylon/-/babylon-6.16.3.tgz",
-      "integrity": "sha1-wpN4E6ify155oABi/EqLFD5yN7s=",
+    "@babel/code-frame": {
+      "version": "7.10.4",
+      "resolved": "https://a0us.jfrog.io/a0us/api/npm/npm/@babel/code-frame/-/code-frame-7.10.4.tgz",
```
Comment (6footGeek) [inline package-lock.json:1521]: @mcastany what do you want to do about the jfrog references?

**Review by 6footGeek — APPROVED** [review]
LGTM apart from the jfrog but can resolve in next release

File: package-lock.json (hunk 2 above)
Comment (mcastany) [reply package-lock.json:1521]: damn, we need to update them

Comment (mcastany) [reply package-lock.json:1521]: I'll keep it as is. We'll change it in the future

**Pull request merged**